torch/_dynamo/optimizations/backends.py

```diff
@@ -747,9 +747,16 @@ def torchxla_trivial(subgraph):
 def torchxla_trace_once(subgraph):
     import torch._dynamo.optimizations.torchxla_integration as integration
 
-    model = subgraph.model
-    example_inputs = subgraph.example_inputs
-    return integration.extract_compiled_graph(model, example_inputs)
+    compiled_graph = None
+
+    def fwd(*args):
+        nonlocal compiled_graph
+        model = subgraph.model
```

Comment:
nit: this will keep subgraph permanently alive, whereas previously it could have been GC'ed after compilation. You should del subgraph being done with compilation

Reply:
good idea